### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by florian-richter**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+assignees: ''
+
+---
+
+**Describe The Bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Set up a repository with the following structure '...'
```
Comment (marikaner) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Do you think the example is helpful? I think it might be confusing ¯\_(ツ)_/¯

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+2. Run `sap-cloud-sdk ...` 
+3. Confirm prompt '....'
+4. See error
+
+**Expected Behavior**
+A clear and concise description of what you expected to happen.
```
Comment (marikaner) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "...of what you expect to happen." or even better "... of the expected behavior."

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (marikaner) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Maybe not "happen" but "change". I think what we want is a description of the feature here, right? Maybe we should just say this ;)

Comment (florian-richter) [reply .github/ISSUE_TEMPLATE/feature_request.md]: It's funny that these are the default github issues templates 😄 Will adjust it though

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Used Versions:**
+ - `node -v`: [e.g. v11.12.0]
+ - `npm -v`: [e.g. 6.12.0]
+ - `sap-cloud-sdk -v`: [e.g. sap-cloud-sdk-cli/0.0.1 darwin-x64 node-v11.12.0]
```
Comment (mr-flannery) [inline .github/ISSUE_TEMPLATE/bug_report.md]: It might make sense in the future to add some thing like a `diagnose` command to the CLI so that users don't have to collect all the versions manually but can run something like `cloud-sdk-cli diagnose` and simply copy the output here.
That's for the future, though.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+
+A clear and concise description of what a solution could look like.
+
+**Describe alternatives you've considered**
+
+A clear and concise description of any alternatives you've considered.
+
```
Comment (mr-flannery) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I think it's worth thinking about marking this section as "optional". IMO this should be communicated like "dear user, if you have thought about several options to approach this problem, we'd love to hear about it". However, personally I wouldn't necessarily expect users to do this. Also weighing options against one another typically requires some amount of understanding of the existing implementation and general constraints, which I also wouldn't expect a user to have. 
However, I'm not sure how other projects handle this, so if this is common practice, feel free to disregard my comment.

**TL;DR:** I'd personally mark this as "optional" to keep the threshold for feature requests low.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (florian-richter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I agree but maybe it's not necessary because the node version is already included in the `-v` command and the npm version might not be relevant. We should evaluate once we gathered some debugging experience.
